### PR TITLE
make test for useres controller and model

### PR DESCRIPTION
**Pull request by kuranovandry**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -26,6 +26,9 @@ gem 'puma'
 gem 'angularjs-rails'
 gem 'angular-ui-bootstrap-rails'
 gem 'angularjs-rails-resource', '~> 2.0.0'
+gem 'sass-rails', '>= 3.2'
+gem "minitest"
```
Comment (maksymst) [inline Gemfile:30]: Why do you need this gem?

File: config/environments/test.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@
   config.eager_load = false
 
   # Configure static asset server for tests with Cache-Control for performance.
-  config.serve_static_assets  = true
+  config.serve_static_files  = true
```
Comment (maksymst) [inline config/environments/test.rb:16]: Why do you need this setting?

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  describe 'validation' do
+    let(:user) { create(:user) }
+    let(:invalid_user) { build(:user, email: nil) }
+    context 'when user provided email' do
+      it 'should pass' do
+        expect(user).to be_valid
+      end
+    end
+
+    context 'when email are missing' do
```
Comment (maksymst) [inline spec/models/user_spec.rb:13]: same as line 21

Comment (maksymst) [inline spec/models/user_spec.rb:13]: when email IS missing

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe User do
+  describe 'validation' do
+    let(:user) { create(:user) }
+    let(:invalid_user) { build(:user, email: nil) }
+    context 'when user provided email' do
```
Comment (maksymst) [inline spec/models/user_spec.rb:7]: instead write this 
`it 'has valid factory' do
    expect(build(:user)).to be_valid
  end`

File: Gemfile (hunk 1 above)
Comment (kuranovandry) [inline Gemfile:30]: Without this gem rspec gives an error

File: config/environments/test.rb (hunk 2 above)
Comment (kuranovandry) [inline config/environments/test.rb:16]: It fix Deprecation Warning

File: Gemfile (hunk 1 above)
Comment (maksymst) [inline Gemfile:30]: What error?

File: config/environments/test.rb (hunk 2 above)
Comment (maksymst) [inline config/environments/test.rb:16]: what kind of warning? Please be specific

File: Gemfile (hunk 1 above)
Comment (kuranovandry) [inline Gemfile:30]: Warning: you should require 'minitest/autorun' instead.

File: config/environments/test.rb (hunk 2 above)
Comment (kuranovandry) [inline config/environments/test.rb:16]: The deprecation warning is shows that serve_static_assets is an old name, and the new name is serve_static_files. Rails makes steady improvements to names and methods, so this is a pretty common warning.

**Pull request merged**